src/hypothesis/views/citation_view.py

```diff
@@ -12,7 +13,7 @@
 from discussion.reaction_models import Vote as GrmVote
 
 
-class CitationViewSet(ModelViewSet, ReactionViewActionMixin):
+class CitationViewSet(ModelViewSet, ReactionViewActionMixin, RhCommentThreadViewMixin):
```

Comment:
I recently learned something new about Python inheritance: 
<img width="664" alt="image" src="https://user-images.githubusercontent.com/364060/224151689-375a93c3-0690-45bf-9e0b-a5bdeb40c48b.png">

So we should correct the ordering of mixins and views.